source/simulator/solver_schemes.cc

```diff
@@ -179,6 +179,20 @@ namespace aspect
                                         dummy);
           break;
         }
+
+        case Parameters<dim>::AdvectionFieldMethod::static_field:
```

Comment:
while you're already here, would you mind adding an empty line also in front of the previous `case` statement in line 166 to make this easier to read?